interface/main/tabs/main.php

```diff
@@ -266,3 +266,24 @@ function isEncounterLocked( encounterId ) {
     font-size: 18px; /* Increase font size */
     }
 </style>
+<script>
+$("#updater-iframe").iziModal({
+           title: '<i class="fa fa-refresh"></i> <?php echo xlt("Updater"); ?>',
+           subtitle: '<?php echo xlt("Configuration settings for updater"); ?>',
+           headerColor: '#F69600',
+           closeOnEscape: true,
+           fullscreen:true,
+           overlayClose: false,
+           closeButton: true,
+           theme: 'dark',  // light
```

Comment:
There are two themes?

Reply:
yes

Reply:
can this be put in external CSS, putting the theme stuff inside the PHP is not optimal.

Reply:
Those are themes for Izimodal @tmccormi . Just options for the modal  code.

Reply:
yes as @teryhill said these are the options for initialising the iziModal @tmccormi

Reply:
I didn't asked what they were, I was curious why they are in the PHP code when, best practice, would say they should be in CSS file that is included.   Not sying it's wrong just asking whay it was done this way.

Reply:
But it is javascript code and cant be put inside css file :confused 
Also this JavaScript code is going to use the php variables present in that page once the updater is complete and also has some work like showing updater notifications